pocketimdb/movies/paginations.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+DEFAULT_PAGE = 1
+
+class CustomPageNumberPagination(PageNumberPagination):
+    page = DEFAULT_PAGE
```

Comment:
cemu ovo sluzi, nisam nasao da umamo opciju da to definisemo nad PageNumberPaginataion https://www.django-rest-framework.org/api-guide/pagination/#pagenumberpagination

Reply:
Postoji property `page` u okviru `PageNumberPagination`  klase, ovo je za slucaj da page query parametar nije zadat, onda ce uzeti stranicu 1 i nju vratiti u odgovoru.

Reply:
Ne vidim ovdje na dokumentaciji, ali ne bi me cudilo da su zakopali negdje

Reply:
Ja sam u samoj klasi to video, isto kao sto u response-u koristim `self.page`. To je isto polje

Reply:
Sad vidim da bez ovoga radi, ocigledno da on ignorise taj moj page sto postavim. Kada kazem bez ovoga radi, mislim na slucaj kada page nije zadat, on po default-u uzima 1.
Onaj primer za custom pagination me je dobrano zavarao..

Reply:
taj sam nasao da je priblizan, nije full verfied, tako da takve primjere sve sa rezervom prolazi